staff/templates/staff.html

```diff
@@ -2,15 +2,14 @@
 
 {% block content %}
 
-
 <div class="main-container">
 
   {% include "message.html" %}
 
     <h1 style="text-align: center"> Staff User Home</h1><br>
     {% if announcements %}
       <div style="text-align: center">
-        <div style="width: 50%; display: inline-block" class="mui-panel">
+        <div style="display: inline-block" class="mui-panel">
```

Comment:
Follow-up in #36 to refactor all of this style="…" stuff.

Reply:
Will reference #36 for this

Reply:
Will reference #36